substreams/src/lib.rs

```diff
@@ -125,6 +125,7 @@ mod state;
 
 pub mod key;
 pub mod store;
+pub mod parser;
```

Comment:
I don't like that it's called parser, `parser` doesn't say anything about what is being parsed.

Second, let's not publicly export the module, remove `pub`. We should only publicly expose the minimum we need for the library to be useful.